src/Factories/TwigFactory.php

```diff
@@ -31,7 +31,10 @@ public function __construct(
 	}
 
 	private function getLoader(): LoaderInterface {
-		if ( !empty( $this->config['loaders']['filesystem'] ) ) {
+		if (
+			is_array( $this->config['loaders'] ) &&
+			isset( $this->config['loaders']['filesystem'] )
+		) {
```

Comment:
You should be able to evaluate using only isset here as it'll fail if any of the levels are missing.

```php
if ( isset( $this->config[ 'loaders' ][ 'filesystem' ] ) ) {
```

Reply:
If I do as you suggest then the initial error comes back:

 ------ --------------------------------------------- 
  Line   src/Factories/TwigFactory.php                
 ------ --------------------------------------------- 
  35     Cannot access offset 'filesystem' on mixed.  
         🪪  offsetAccess.nonOffsetAccessible         
 ------ --------------------------------------------- 
 
 What should be the next step then?